### PR TITLE
Prepare mint

### PR DESCRIPTION
**Pull request by 8ball030**

### CODE REVIEW

File: packages/eightballer/agents/liquidation_station/aea-config.yaml [hunk 1]
```diff
@@ -80,8 +81,8 @@ type: skill
 models:
   params:
     args:
-      infura_api_key: ${SKILL_LIQUIDATION_STATION_MODELS_PARAMS_ARGS_INFURA_API_KEY:str}
-      polygonscan_api_key: ${SKILL_LIQUIDATION_STATION_MODELS_PARAMS_ARGS_POLYGONSCAN_API_KEY:str}
+      infura_api_key: ${str:000000}
+      polygonscan_api_key: ${str:000000}
```
Comment (Karrenbelt) [inline packages/eightballer/agents/liquidation_station/aea-config.yaml:85]: is this intentional?

**Comment by 8ball030** [comment]
Yup

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: packages/packages.json [hunk 2]
```diff
@@ -31,40 +28,29 @@
...
         "protocol/valory/acn/1.1.0": "bafybeif2th25e4gaejgpg373wal4bkpyfxdbee7f7gbrb64vrgrwhbakxm",
         "connection/valory/p2p_libp2p_client/0.1.0": "bafybeidfm65eece533hfvg2xyn4icpmvz4lmvbemstrlo3iuffb7e72ycq",
         "contract/valory/multisend/0.1.0": "bafybeicj5ob6zekorswg6j6uf7rg2ft56so4kulkevpcuzxyhww6gnh4xq",
         "protocol/valory/ipfs/0.1.0": "bafybeidjn5fa2tjah3jif4tt67rccvtflkmkj5gnjq2jssdkxdzwrzzasy",
         "connection/valory/ipfs/0.1.0": "bafybeifo4tzgjtkt422ks3ooipum7p4ycr6rrdtuab246owo2r74vmseee"
-    }
-}
+    },
+    "third_party": {}
+}
```
Comment (Karrenbelt) [inline packages/packages.json:56]: how come we don't need to list any third party components anymore?

**Comment by 8ball030** [comment]
Dude it's a pr, please don't review it unless I ask I'm working on it

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**